ci/citest/acceptance-test/tests/local_image_test.go

```diff
@@ -3,20 +3,46 @@
 package tests
 
 import (
+	"fmt"
+	"path/filepath"
 	"strings"
 	"testing"
 	"time"
 )
 
 func TestLocalImage(t *testing.T) {
 
+	stdout, _, err := RunSsh(scheduler_ip, fmt.Sprintf("[ -f /var/www/html/images/centos/7/amd64/meta.tar.xz ] && echo meta.tar.xz found || echo meta.tar.xz not found"))
+
+        if err != nil {
+                t.Error(err)
+        }
+
+        t.Log(stdout.String())
+
 	// Use custom lxc-template.
-	stdout, _ := RunCmdAndReportFail(t, "openvdc", "run", "centos/7/lxc", `{"lxc_template":{"template":"openvdc"}, "node_groups":["linuxbr"]}`)
+	stdout, _ = RunCmdAndReportFail(t, "openvdc", "run", "https://raw.githubusercontent.com/axsh/openvdc/template-misc/templates/centos/7/lxc2.json", `{"node_groups":["linuxbr"]}`)
 	instance_id := strings.TrimSpace(stdout.String())
 
-	_, _ = RunCmdAndReportFail(t, "openvdc", "show", instance_id)
 	WaitInstance(t, 10*time.Minute, instance_id, "RUNNING", []string{"QUEUED", "STARTING"})
-	RunSshWithTimeoutAndReportFail(t, executor_lxc_ip, "sudo lxc-info -n "+instance_id, 10, 5)
+
+	configFile := filepath.Join("/var/lib/lxc/", instance_id, "config")
+	stdout, _, err = RunSsh(executor_lxc_ip, fmt.Sprintf("echo | sudo head -n 1 %s", configFile))
+
+	if err != nil {
+		t.Error(err)
+	}
+	if stdout.Len() == 0 {
+		t.Errorf("Couldn't read %s", configFile)
+	}
+
+	t.Log(stdout.String())
```

Comment:
Better to check if verbose flag is passed from command line when you put non-error messages.

```
if testing.Verbose() {
    t.Log(stdout.String())
}
```